### ISSUE TITLE
Occasionally, Microweber's favicon shows instead of the configured or whitelabeled favicon.

### ISSUE THREAD

**Issue by RogerAntonio** [microweber/microweber#907]
On my website, I have my Favicon added to my Settings > General Settings and my logo is also added to my whitelabel settings.
![image](https://user-images.githubusercontent.com/5510591/179509611-31aca8ca-6abc-4c31-b3e0-72e81c3b4483.png)

However, MW's logo seeps through in [MailChimp](https://mailchimp.com)'s Editor when inserting my url into a mailing template.
![image](https://user-images.githubusercontent.com/5510591/179508855-7e194a1b-7527-4148-b7a8-947c3acdf920.png)

**Comment by peter-mw** [comment]
hi, you can replace the favicon.ico file in the website root 

seems the mailchimp is using favicon.ico file for the favicon tag

**Comment by RogerAntonio** [comment]
Okay, so it doesn't get replaced with new updates then?

**Comment by peter-mw** [comment]
No, its not replaced on update by standalone updater here are all files that get replaced https://github.com/microweber-modules/standalone-updater/blob/main/src/standalone-installation-setup/actions.source#L235-L254
